core/arch/x86/instr_create.h

```diff
@@ -2303,6 +2302,8 @@
 #define INSTR_CREATE_maxss(dc, d, s) instr_create_1dst_2src((dc), OP_maxss, (d), (s), (d))
 #define INSTR_CREATE_maxpd(dc, d, s) instr_create_1dst_2src((dc), OP_maxpd, (d), (s), (d))
 #define INSTR_CREATE_maxsd(dc, d, s) instr_create_1dst_2src((dc), OP_maxsd, (d), (s), (d))
+#define INSTR_CREATE_pshufb(dc, d, s) \
```

Comment:
nit: pshufb is SSSE3 (right?) so best to put it with the SSSE3 group down below on line 2358+ (see the comment there)